.github/workflows/ci.yaml

```
@@ -341,23 +341,22 @@ jobs:
           git checkout v${{ matrix.from_version }} --no-overlay -- tests/helper_components
           # Remove examples from the workspace because they may no longer exist or their Cargo.toml might prevent the build of the updater
           sed -i "/examples/d" Cargo.toml
-    - name: "Commit old checkout"
-      run: |
-          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-          git config --global user.name "${GITHUB_ACTOR}"
-          git commit -a -m "REVERT TESTS TO v${{ matrix.from_version }}"
-    - name: run the updater
-      run: |
-        cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i examples/*/*.60
-        cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i examples/*/*/*.60
-        cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i tests/cases/*.60
-        cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i tests/cases/*/*.60
-    - name: Show the diff
-      run: git diff
+    # - name: "Commit old checkout"
+    #   run: |
+    #       git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+    #       git config --global user.name "${GITHUB_ACTOR}"
+    #       git commit -a -m "REVERT TESTS TO v${{ matrix.from_version }}"
+    # - name: run the updater
+    #   run: |
+    #     cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i examples/*/*.slint
+    #     cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i examples/*/*/*.slint
+    #     cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i tests/cases/*.slint
+    #     cargo run --bin syntax_updater -- --from ${{ matrix.from_version }} -i tests/cases/*/*.slint
+    # - name: Show the diff
+    #   run: git diff
     - name: test
       uses: actions-rs/cargo@v1
       with:
           command: test
-          # skip example_printerdemo_old because that was not in 0.0.5
           # skip test_interpreter_examples_rotate because the internal rotate element was removed
-          args: --bin test-driver-interpreter -- --skip example_printerdemo_old --skip test_interpreter_examples_rotate
+          args: --bin test-driver-interpreter -- --skip test_interpreter_examples_rotate
```

```suggestion
          args: --bin test-driver-interpreter
```

no need to skip anything.  the rotate example was updated to the new API in 0.3  (comment can also go)